lib/ansible/modules/web_infrastructure/jenkins_script.py

```diff
@@ -142,24 +165,35 @@ def get_crumb(module):
 
 def main():
 
+    argument_spec = url_argument_spec()
```

Comment:
This makes sense to switch to ... but for compatibility and simplicity user and password should be made aliases of url_username and url_password ... and if using the framework then the notes should make it clear that all params supported by it are possible.

Reply:
Yes, that makes sense. Looks like `user` and `password` are a couple of other parameters that diverged from `jenkins_plugin`, so I will try to bring some uniformity to this.